tinygrad/tensor.py

```diff
@@ -1631,6 +1631,41 @@ def argmin(self, axis=None, keepdim=False):
     """
     return (-self).argmax(axis=axis, keepdim=keepdim)
 
+  @staticmethod
```

Comment:
This should not be staticmethod. It shouldn't support a list of Tensor, only one tensor called `self`